config/locales/views/assignments/en.yml

```diff
@@ -28,8 +28,11 @@ en:
     hidden: "%{assignment_text} (hidden)"
     manage_course_work: Manage Course Work
     marking_scheme: 'Marking Scheme: %{identifier}'
+    maximum_file_size: 'Maximum File Size: %{file_size} MB'
```

Comment:
Move this key to `views/submissions/en.yml` under `submissions -> student`.

Change text to `'Maximum file upload size: %{file_size} MB'`